roles/installer/tasks/database_configuration.yml

```diff
@@ -85,21 +85,22 @@
 
 - name: Set PostgreSQL Configuration
   set_fact:
-    pg_config: '{{ _generated_pg_config_resources["resources"] | default([]) | length | ternary(_generated_pg_config_resources, _pg_config) }}'
-  no_log: "{{ no_log }}"
```

Comment:
I think this removal is unintended. It results in invalid syntax:
* https://github.com/ansible/awx-operator/blob/6dbe2aa134d1a4ebd03a70bc584921eff54914d6/roles/installer/tasks/database_configuration.yml#L86-L89

Reply:
That was not intended.I updated the PR set_fact got added twice in the same task.